test/example.yaml

```diff
@@ -8,6 +8,7 @@ templates:
   - name: "example-cluster"
     driver: podman
     container_runtime: cri-o
+    addons: 'ingress'
```

Comment:
This must be list, try:

    addons:
      - ingress

But, do we need the ingress addon if we deploy ingress later?

I would try to add the components in separate commits to make debugging
easier, and instead of replacing busybox, add nginx:

- commit 1: deploy nginx
- commit 2: configure ingress so we can access nginx
- commit 3: expose the busybox log via nginx?

This sounds like first step in having cooler example, being able to see the
application log in a local browser.

Later we can replace busybox with more interesting application.

Reply:
Looking at the docs this seems to be correct (using ingress addon + ingress yaml):
https://kubernetes.io/docs/tasks/access-application-cluster/ingress-minikube/